main.scss

```diff
@@ -26,7 +25,45 @@
 @import "src/scss/themes/package";
 @import "src/scss/themes/live";
 
-@import "src/scss/teaser";
+/// Output styles for teasers.
+/// Can be passed a list of styles to output,
+/// or 'all' to output everything
+///
+/// @param {List} [all] $elements
+/// @param {List} [all] $themes
+@mixin oTeaser($elements: 'all', $themes: 'all') {
```

Comment:
How come this public mixin takes two parameters instead of an options `$opts` map? I kind of thought all our main component mixins had the same kind of interface where they took a map object.

Reply:
Will follow up with more changes :)